refinery/data_set_manager/utils.py

```diff
@@ -780,7 +783,7 @@ def is_field_in_hidden_list(field):
                      'assay_uuid', 'type', 'is_annotation', 'species',
                      'genome_build', 'name', 'django_ct']
 
-    if field in hidden_fields:
+    if field in hidden_fields or NodeIndex.GENERIC_SUFFIX in field:
```

Comment:
@jkmarx : This is the part that really matters: The chunk above is just using the constant more consistently, but doesn't change behavior.